run.py

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env python3
+import logging
+import json
+import os, os.path as op
+import subprocess as sp
+import sys
+import zipfile
+from collections import OrderedDict
+import psutil
+
+import flywheel
+from utils import *
+from custom_gear_utils import *
```

Comment:
Best practice to avoid `from package import *`. `import package` is more readable and avoids import conflicts.